modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/EmbeddedScene.java

```diff
@@ -231,8 +231,8 @@ public boolean getPixels(final IntBuffer dest, final int width, final int height
             {
                 return false;
             }
-            scaledWidth = Math.round(scaledWidth * texScaleFactorX);
-            scaledHeight = Math.round(scaledHeight * texScaleFactorY);
+            scaledWidth = (int) Math.ceil(scaledWidth * texScaleFactorX);
+            scaledHeight = (int) Math.ceil(scaledHeight * texScaleFactorY);
```

Comment:
Since this is in code that is shared by all embedded scenes, have you verified that FXCanvas (SWT interop) doesn't need similar changes?